src/app/containers/StoryPromo/IndexAlsos/index.test.jsx

```diff
@@ -72,5 +72,47 @@ describe('Index Alsos', () => {
       const headline = secondListItem.getElementsByTagName('span')[0].innerHTML;
       expect(headline).toEqual('Overtyped headline');
     });
+
+    it('should render a promo headline', () => {
```

Comment:
I'd put these new assertions into their own describe block to make clear they are testing functionality specific to `LINK/PRO` 'indexalsos` rather than `STY` index alsos.

Reply:
Done